sklearn/base.py

```diff
@@ -1087,3 +1087,10 @@ def is_outlier_detector(estimator):
         True if estimator is an outlier detector and False otherwise.
     """
     return getattr(estimator, "_estimator_type", None) == "outlier_detector"
+
+
+def not_covered_by_test():
+    """Function not covered by tests.
+    It is used to test the coverage report generation.
```

Comment:
I think this should fix [the remaining error on the CI](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=53166&view=logs&j=78a0bf4f-79e5-5387-94ec-13e67d216d6e&t=f1857171-4a53-55c7-3ab5-90acfe091baa&l=615).
```suggestion
    """Function not covered by tests.

    It is used to test the coverage report generation.
```